### PR TITLE
feature/#46-Create-Doughnut-chart-for-showing-the-services-called-on-the-Dashboard-Frontend

### PR DESCRIPTION
**Pull request by ChristophBaierATB**

### CODE REVIEW

File: spring-boot/admin/src/main/resources/messages.properties [hunk 1]
```diff
@@ -25,6 +25,7 @@ admin.dashboard.stats.errors.desc=Number of API calls failed in the selected dat
 admin.dashboard.stats.http.response.codes.desc=HTTP response codes in the selected date range.
 admin.dashboard.num.errors.desc=Total amount of errors in the selected date range.
 admin.dashboard.stats.calls.api.user.desc=Calls api user in the selected date range.
+admin.dashboard.stats.calls.per.service.desc=Shows the number of calls per service in the selected data-range
```
Comment (ChristophBaierATB) [inline spring-boot/admin/src/main/resources/messages.properties:28]: Please Christoph, change this value to

**Review by ChristophBaierATB — COMMENTED** [review]
Values to change.

File: spring-boot/admin/src/main/resources/templates/admin/dashboard.html [hunk 2]
```diff
@@ -92,6 +92,7 @@
                     loadStatisticsPerApiConsumerInDescendingOrderByNoOfCalls(dateRangeUrlSuffix);
                     loadChart('/admin/dashboard/httpResponseStatsChart' + dateRangeUrlSuffix, 'httpStatsChartCanvas');
                     loadChart('/admin/dashboard/userStatsDoughnutChart' + dateRangeUrlSuffix, 'userStatsChartCanvas');
+                    loadChart('/admin/dashboard/userStatsDoughnutChart' + dateRangeUrlSuffix, 'callsPerServiceChartCanvas');
```
Comment (BrunoSilvaATB) [inline spring-boot/admin/src/main/resources/templates/admin/dashboard.html:95]: @ChristophBaierATB please change this to `loadChart('/admin/dashboard/serviceStatsChartCanvas ' + dateRangeUrlSuffix, 'callsPerServiceChartCanvas');`

**Review by BrunoSilvaATB — CHANGES_REQUESTED** [review]
Christoph, please proceed to apply these changes so that I can start the merging process with the backend.

**Review by BrunoSilvaATB — APPROVED** [review]
The code is now approved and will be merged to the Backend code.

**Pull request closed without merging**